dkist/io/asdf/converters/tiled_dataset.py

```diff
@@ -5,16 +5,23 @@ class TiledDatasetConverter(Converter):
     tags = [
         "tag:dkist.nso.edu:dkist/tiled_dataset-0.1.0",
         "asdf://dkist.nso.edu/tags/tiled_dataset-1.0.0",
+        "asdf://dkist.nso.edu/tags/tiled_dataset-1.1.0",
     ]
     types = ["dkist.dataset.tiled_dataset.TiledDataset"]
 
     def from_yaml_tree(cls, node, tag, ctx):
         from dkist.dataset.tiled_dataset import TiledDataset
 
-        return TiledDataset(node["datasets"], node["inventory"])
+        try:
+            mask = node["mask"]
+        except KeyError:
+            mask = None
+
+        return TiledDataset(node["datasets"], node["inventory"], mask)
 
     def to_yaml_tree(cls, tiled_dataset, tag, ctx):
         tree = {}
         tree["inventory"] = tiled_dataset._inventory
         tree["datasets"] = tiled_dataset._data.tolist()
+        tree["mask"] = tiled_dataset._data.mask.tolist()
```

Comment:
I worry that doing this will add more yaml to the file, when most of the time we probably don't need it, i.e. it's the same as "where datasets is not None". I am not sure if it's worth even trying to investigate this or not.

Reply:
Following more discussion in #500 I think we should just make this a numpy array and not a list.